runtime/gateway.go

```diff
@@ -169,18 +173,25 @@ func (gateway *Gateway) Bootstrap(register RegisterFn) error {
 	}
 
 	// start TChannel server
-	//addr := backend.IP + ":" + strconv.Itoa(int(backend.Port))
-	//ln, err := net.Listen("tcp", addr)
-	//if err != nil {
-	//	return err
-	//}
-	//
-	//realAddr := ln.Addr().(*net.TCPAddr)
-	//backend.RealPort = int32(realAddr.Port)
-	//backend.RealAddr = realAddr.IP.String() + ":" + strconv.Itoa(int(backend.RealPort))
-	//
-	//// tchannel serve does not block, connection handling is done in different goroutine
-	//err = backend.Channel.Serve(ln)
+	// TODO: proper IP
+	tchanAddr := "127.0.0.1:" + strconv.Itoa(int(gateway.TChanPort))
```

Comment:
tchannel should generally never listen on 127.0.0.1; we already have a library somewhere for finding the correct external IP.